src/collectors/postgres/postgres.py

```diff
@@ -54,9 +76,9 @@ def get_default_config(self):
         Return default config.
         """
         config = super(PostgresqlCollector, self).get_default_config()
+
         config.update({
             'path': 'postgres',
-            'host': 'localhost',
```

Comment:
I already added it in my last commit in the `collect` method. Same way as pgbouncer collector.